.github/workflows/build.yml

```diff
@@ -140,6 +140,12 @@ jobs:
 
           CONFIG_WIFI_NETWORK='test' CONFIG_WIFI_PASSWORD='password' laze build --global --partition hash:${{ matrix.partition }}
 
+      # cleanup previous build artifacts. do in background (no need to wait).
+      - name: "cleanup stable build artifacts"
+        if: github.event_name == 'schedule' && steps.result-cache.outputs.cache-hit != 'true'
+        run: |
+          mv build build.tmp && rm -rf build.tmp &
```

Comment:
Those not too familiar with shell operator precedence (such as me) can easily convince themselves that this still waits for the moving to complete by running `sleep 10 && echo foo &`.